Userland/Libraries/LibGfx/ImageFormats/TIFFLoader.cpp

```diff
@@ -43,6 +43,14 @@ class TIFFLoadingContext {
         return {};
     }
 
+    ErrorOr<void> ensure_conditional_tags_presence() const
```

Comment:
s/presence/present/

Reply:
That looks really wrong to me, are you sure?

Reply:
Yes. "Ensure the tags are present".

Reply:
I still don't understand why this is better than "Ensure tag presence" but ok